scripts/local/versions.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+# Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+# See the file LICENSE for licensing terms.
+
+# Set up the versions to be used
+AWM_RELAYER_VERSION=${AWM_RELAYER_VERSION:-'v0.2.1'}
+SUBNET_EVM_VERSION=${SUBNET_EVM_VERSION:-'v0.5.4'}
```

Comment:
You're probably already on top of this, but we'll need to update this to `v0.5.6` with the latest commits from `main`.

Reply:
Yup, just waiting on https://github.com/ava-labs/teleporter/pull/29